lib/generators/data_script/data_script_generator.rb

```diff
@@ -0,0 +1,9 @@
+class DataScriptGenerator < Rails::Generators::NamedBase
```

Comment:
Adding a generator to make it easy to create a data script and a corresponding spec - this is common enough where we should automate it and keep it safe by default.